spec/spec.yaml

```diff
@@ -25,6 +25,15 @@ openjdk-11:
   filename:   'Main.java'
 java:
   use:        'openjdk-11'
+
+kotlin:
+  image: 'schlaubiboy/kotlin:1.5.10'
+  filename: 'main.kt'
+  entrypoint: 'kotlinc main.kt && java MainKt'
+kotlin-script:
+  image: 'schlaubiboy/kotlin:1.5.10'
+  filename: 'main.kts'
+  entrypoint: 'kotlinc -script main.kts'
```

Comment:
The file name must not be passed when using `entrypoint`, because the value of the `filename` spec property is appended to the value of the `entrypoint` command definition. See [spec definition](https://github.com/ranna-go/ranna/wiki/%E2%9A%99-Spec#spec-model).